<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>173</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Create/Update Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250,243,243)</string>
  </property>
  <widget class="QLabel" name="uName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="stdyNameLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Case Study</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipFolderLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Select Inphase folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="opFolderLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Select Outphase folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);
background-color: rgb(68, 68, 68);
color: rgb(184, 184, 184);


color: rgb(0,0,0);
background-color: rgb(167,187,199)
</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uNameDat">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="stdyNameDat">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ipBrowse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);
background-color: rgb(68, 68, 68);
color: rgb(184, 184, 184);


color: rgb(0,0,0);
background-color: rgb(167,187,199)
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="opBrowse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);
background-color: rgb(68, 68, 68);
color: rgb(184, 184, 184);


color: rgb(0,0,0);
background-color: rgb(167,187,199)
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="uNameCmb">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);


color: rgb(0, 0, 0);
background-color: rgb(167,187,199);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="createRdBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="updtRdBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(21, 21, 21);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(42, 42, 42);

color: rgb(184, 184, 184);


color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
